### PR TITLE
Add basic model conversion support

### LINKED ISSUE #211: 3D models synced with open layers?
I saw #163 . What I'm trying to do:
1. Make a feature, like a point, in Openlayers.
2. Set a key value pair with ol.Feature.set('model', '/models/airplane.gltf'). Maybe with some basic options like minimumPixelSize and scale.
3. When switching from Openlayers to Cesium, it replaces the point with that model.

Is something like having a 3D model synced (i.e. position) with a point in Openlayers possible? If so, can I please get an example? Thank you.

### PR DESCRIPTION
**Pull request by gberaudo**
Closes #211.
Related to #435.

image

### CODE REVIEW

File: examples/vectors.js [hunk 1]
```diff
@@ -44,6 +46,14 @@ const cervinFeature = new ol.Feature({
 cervinFeature.getGeometry().set('altitudeMode', 'clampToGround');
 
 
+const modelFeatures = [-1, -1 / 2, 0, 1 / 2, 1, 3 / 2].map(
+    factor => new ol.Feature({
+      geometry: new ol.geom.Point([852641, 5776749, 4500]),
+      'rotation': factor * Math.PI
```
Comment (fredj) [inline examples/vectors.js]: remove quotes?

Comment (gberaudo) [reply examples/vectors.js]: I like to be ready for GCC compilation of the examples :P

**Comment by gberaudo** [comment]
@fredj, as discussed yesterday, I changed the host of the `olcs_model` object to be an ol.Object.
Does it look good to you?

**Comment by gberaudo** [comment]
Thanks for the rewiev @fredj.

**Pull request merged**